api-catalog-package/src/main/resources/bin/start.sh

```diff
@@ -56,14 +56,16 @@ else
 fi
 echo "jar file: "${JAR_FILE}
 # script assumes it's in the catalog component directory and common_lib needs to be relative path
-
 if [ -z "${CMMN_LB}" ]
 then
     COMMON_LIB="../apiml-common-lib/bin/api-layer-lite-lib-all.jar"
 else
     COMMON_LIB=${CMMN_LB}
 fi
 
+# script assumes it's in the api-catalog component directory and jvm.security.override.properties needs to be relative path
+JVM_SECURITY="../apiml-common-lib/bin/jvm.security.override.properties"
```

Comment:
Should we set this based on the type of keyring? Is the IBMJCACCA provider always available?

Reply:
Also another concern, what if the customer has a custom list already in place for some reason, could this be a potential breaking change? 
I think we should provide a way use the JVM's default (for support reasons)